ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
(purecss is a different problem than splitting main.css. Here, I answer only about splitting main.css)

Styling the home page cards (using purecss) toke me very few time. Include purecss file, add tags in the html and done ... almost.

What took me time it to check that everything still work properly and that I've broke nothing.
As all rules were in the same css file, there were (and probably are) conflicts between them and some rules were rewriting other.
Few example:
- The  `.library` rules from monitoring where changing the font color to orange. But the rules `.grid .library` was resetting it.
- The rules for home cards background were not consistent, for blog (and two others), the rules was `.grid .blog` instead of just `.blog`. This was done to not conflict with the blog application where we use "card .blog" to display a blog article.

In conclusion, I spent a lot of time to check page by page that I break nothing. And it take a lot of time.
By splitting css in separate files included only when necessary, we avoid a lot of potential conflict and then simplify a lot our (future) work.

On serving those files:
- We replace 1 big css file by 2 smaller ones. There is indeed one connection more, but less data transferred. At worst a user will end by having all the css downloaded but it will no be at once.
- The static cache made by bochecha will greatly help here.
- In case of update of a application css, only this specific css is updated, all the others do not change and so are not downloaded again.

Associated with the assurance that if I change only the library.css file, only the library application will be impacted and that I don't have to check other application, I think it's worth it.